.gitlab-ci.yml

```diff
@@ -273,7 +273,7 @@ spellcheck:
     - git fetch origin +${CI_DEFAULT_BRANCH}:${CI_DEFAULT_BRANCH}
     - echo "___Spellcheck is going to check your diff___"
     - cargo spellcheck list-files -vvv $(git diff --diff-filter=AM --name-only $(git merge-base ${CI_COMMIT_SHA} ${CI_DEFAULT_BRANCH} -- :^bridges))
-    - time cargo spellcheck check -vvv --cfg=scripts/gitlab/spellcheck.toml --checkers hunspell --code 1
+    - time cargo spellcheck check -vvv --cfg=.spellcheck/spellcheck.toml --checkers hunspell --code 1
```

Comment:
Is it necessary to move this file to the `.spellcheck` folder? Recently we unified the structure of CI files in all major repos (https://github.com/paritytech/ci_cd/issues/313) and would like to keep all files in one place.

Reply:
Yes, I was opposing the change to move them out of there from the get go, since that's the default discovery location of `cargo-spellcheck` and avoids yet another arg required.

Reply:
arg is not that bad, but what if someone wants several config files for the different crates, dirs, and languages?

Reply:
btw you still use the extra arg here

Reply:
Yes, I use the extra arg here for explicitness. But for the average user I'd not want that, since the output would be an excessive amount of detected errors, that are covered by `lingua.dic`.